dougUCN/LANE
# Add GraphQL Code Generator

Adds typed GraphQL code generation for urql.

### Testing Steps
Ensure that a `generated.ts` file is created in the `src` directory after running `npm run generate`.  
Types should now show up when hovering over declared variables that are associated with GraphQL.
